docs/book/attributes.rst

```diff
@@ -0,0 +1,87 @@
+.. index::
+   single: Attributes
+
+Attributes
+==========
+
+Attributes in Sylius are used to describe traits shared among entities. The best example are products, that may be of
+the same category and therefore they will have many similar attributes such as **number of pages for a book**,
+**brand of a T-shirt** or simply **details of any product**.
+
+Attribute
+---------
+
+The **Attribute** model has a translatable name (like for instance ``Book pages``), code (``book_pages``) and type (``integer``).
+There are a few available types of an Attribute:
+
+* text (*default*)
+* checkbox
+* integer
+* percent
+* textarea
+
+What these types may be useful for?
+
+- text - brand of a T-Shirt
+- checkbox - show whether a T-Shirt is made of cotton or not
+- integer - number of elements when a product is a set of items.
+- percent - show how much cotton is there in a piece of clothing
+- textarea - display more detailed information about a product
+
+How to create an Attribute?
+---------------------------
+
+To give you a better insight into Attributes, let's have a look how to prepare and add an Attribute with a Product to the system programatically.
+
+.. code-block:: php
+
+   // To assign Attributes to Products firstly you will need a factory for ProductAttributes
```

Comment:
+1, also the example is a bit wrong, as we add same attribute twice to a single product, which is not supported yet. Let's add a single attribute for now!